### PR TITLE
ci: add github actions workflows

### PR DESCRIPTION
**Pull request by tylermarien**
NOTE: There are extra steps to this PR. Please make sure to coordinate in #proj-github-actions before you merge this PR.

#### Why
We are changing all of our repositories to use Github Actions instead of CircleCI. This PR will remove the CircleCI config and add config for Github Actions.

#### What
- Removed .circleci directory
- Added default.yml and security-check.yml workflows

#### Next steps
- Change branch protection rules to use Github Actions checks rather than CircleCI checks
- Merge this PR
- Stop building CircleCI project

#### Questions / Concerns
If you have any questions you can post them in the #proj-github-actions Slack channel.

### CODE REVIEW

File: .github/workflows/default.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# fast feedback. It also gets run when a pull request is created so that we can
+# run the Sonarqube quality gate (which needs information from the PR). Subsequent
+# pushes to the branch will provide PR information of any open PRs.
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+
+concurrency:
+  group: ${{ github.ref }}
```
Comment (cchawn) [inline .github/workflows/default.yml:13]: we need to update this to include the workflow :)

File: .github/workflows/security-check.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Security Check
+
+on:
+  schedule:
+    - cron: '15 11 * * *' # 11:15 am UTC: 6:15 am EST / 7:15 am EDT
+
+concurrency:
+  group: ${{ github.ref }}
```
Comment (cchawn) [inline .github/workflows/security-check.yml:8]: need the workflow name here too

File: ws-schemas-cli-modules/yarn-error.log [hunk 3]
```diff
@@ -0,0 +1,36 @@
+Arguments: 
+  /Users/mkuznetsova/.nvm/versions/node/v14.17.6/bin/node /Users/mkuznetsova/.nvm/versions/node/v14.17.6/bin/yarn global add --global-folder ws-schemas-cli-modules @wealthsimple/ws-schemas-cli@^1.7.5
```
Comment (hannahyeates) [inline ws-schemas-cli-modules/yarn-error.log:2]: should this be included in this PR?

File: ws-schemas-cli-modules/yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,36 @@
+Arguments:
```
Comment (cchawn) [inline ws-schemas-cli-modules/yarn-error.log:1]: I don't think we meant to commit this file?

**Review by hannahyeates — APPROVED** [review]
LGTM

**Review by cchawn — APPROVED** [review]

**Pull request merged**